nncf/tensorflow/sparsity/callbacks.py

```diff
@@ -47,7 +47,6 @@ def _prepare_for_tensorboard(self, stats: NNCFStatistics):
         tensorboard_stats = {
             f'{base_prefix}/sparsity_level_for_model': ms.sparsity_level,
             f'{base_prefix}/sparsity_level_for_sparsified_layers': ms.sparsity_level_for_layers,
-            f'{base_prefix}/target_sparsity_level': stats.target_sparsity_level,
```

Comment:
What is the reason for this change?

Reply:
Thanks for the notice. It is a leftover. Fixed.